api/server/sdk/credentials_test.go

```diff
@@ -598,90 +620,11 @@ func TestSdkCredentialEnumerateAzureSuccess(t *testing.T) {
 		// Setup client
 	c := api.NewOpenStorageCredentialsClient(s.Conn())
 
-	// Enumerate Azure Credentials
-	resp, err := c.EnumerateForAzure(context.Background(), req)
+	// Inspect
+	resp, err := c.Inspect(context.Background(), req)
```

Comment:
can you please add example of how to use Inspect resp here?

Reply:
Yeah, great idea.

Reply:
Actually, there is an example on line 627, but I'll add more info to the proto file docs.